src/Registry/Utils/TermObject.php

```diff
@@ -0,0 +1,322 @@
+				'connectionTypeName' => ucfirst( $tax_object->graphql_single_name ) . 'ToAncestors' . ucfirst( $tax_object->graphql_single_name ) . 'Connection',
+				'resolve'            => function ( Term $term, $args, AppContext $context, $info ) use ( $tax_object ) {
+					if ( ! $tax_object instanceof WP_Taxonomy ) {
+						return null;
+					}
+
+					$ancestor_ids = get_ancestors( absint( $term->term_id ), $term->taxonomyName, 'taxonomy' );
+
+					if ( empty( $ancestor_ids ) ) {
+						return null;
+					}
+
+					$resolver = new TermObjectConnectionResolver( $term, $args, $context, $info, $tax_object->name );
+					$resolver->set_query_arg( 'include', $ancestor_ids );
+
+					return $resolver->get_connection();
+				},
+			];
+		}
+
+		// Used to ensure contentNodes connection doesn't get registered multiple times.
+		$already_registered = false;
+		$allowed_post_types = WPGraphQL::get_allowed_post_types( 'objects' );
+
+		foreach ( $allowed_post_types as $post_type_object ) {
+
+			if ( ! in_array( $tax_object->name, get_object_taxonomies( $post_type_object->name ), true ) ) {
+				continue;
+			}
+
+			// ContentNodes.
+			if ( ! $already_registered ) {
+
+				$connections['contentNodes'] = PostObjects::get_connection_config( $tax_object, [
+					'toType'  => 'ContentNode',
+					'resolve' => function ( Term $term, $args, $context, $info ) {
+						$resolver = new PostObjectConnectionResolver( $term, $args, $context, $info, 'any' );
+						$resolver->set_query_arg( 'tax_query', [
+							[
+								'taxonomy'         => $term->taxonomyName,
+								'terms'            => [ $term->term_id ],
+								'field'            => 'term_id',
+								'include_children' => false,
+							],
+						] );
+
+						return $resolver->get_connection();
+					},
+				] );
+
+				// We won't need to register this connection again.
+				$already_registered = true;
+			}
+
+			// PostObjects.
+			$connections[ $post_type_object->graphql_plural_name ] = PostObjects::get_connection_config( $post_type_object, [
+				'toType'     => $post_type_object->graphql_single_name,
+				'queryClass' => 'WP_Query',
+				'resolve'    => function ( Term $term, $args, AppContext $context, ResolveInfo $info ) use ( $post_type_object ) {
+					$resolver = new PostObjectConnectionResolver( $term, $args, $context, $info, $post_type_object->name );
+					$resolver->set_query_arg( 'tax_query', [
+						[
+							'taxonomy'         => $term->taxonomyName,
+							'terms'            => [ $term->term_id ],
+							'field'            => 'term_id',
+							'include_children' => false,
+						],
+					] );
+
+					return $resolver->get_connection();
+				},
+			]);
+		}
+
+		// Merge with connections set in register_taxonomy.
+		if ( ! empty( $tax_object->graphql_connections ) ) {
+			$connections = array_merge( $connections, $tax_object->graphql_connections );
+		}
+
+		// Remove excluded connections.
+		foreach ( $tax_object->graphql_exclude_connections as $connection_name ) {
```

Comment:
Does it make sense to support `graphql_exclude_connections` at a lower level? 

For example, in the WPObjectType and WPInterfaceType?

For example, we could then do something like this:

```php
apply_filters( 'graphql_wp_object_type_config', function( $config ) { 
  $config['graphql_exclude_connections'] = [ 'someConnectionToExclude'];
  return $config;
});
```

And this functionality could then work for all types, not just types that were added to the graph by `register_post_type` and `register_taxonomy`?

Like, I know some folks using WPGraphQL to get data from external data sources, like shopify, etc. 

If they had some connections registered, and a plugin author wanted to quickly exclude them, they could use the filter above, and it would work for any type, not just Post/Term types. 🤔

Reply:
I suppose even in the situation I brought up, we could just use `apply_filters( 'graphql_wp_object_type_config', ... );` and unset the connections manually as well. 

Maybe not a big deal 🤔

Reply:
I think a whole slew of lower-level filters, along with some additional `(de)register_graphql_*` access methods would be great.

This PR though is about making it easier to for devs to handle auto registering CPTs/Terms _without_ the use of multiple actions/filters.

We can always adjust the underlying logic used to handle the WP_Type class properties in a future PR, and I think that by handling it separately/incrementally we can make a more informed design decision about where these new filters should go, instead of implementing them now and possibly needing to adjust/deprecate them once we have a concrete list of additional use cases/considerations.